static/app/utils/profiling/profile/importProfile.tsx

```diff
@@ -259,13 +258,51 @@ export function importSentryContinuousProfileChunk(
     input.platform
   );
 
+  const samplesByThread: Record<
+    string,
+    Profiling.SentryContinousProfileChunk['profile']['samples']
+  > = {};
+
+  for (let i = 0; i < input.profile.samples.length; i++) {
+    const sample = input.profile.samples[i];
+    if (!samplesByThread[sample.thread_id]) {
+      samplesByThread[sample.thread_id] = [];
+    }
+    samplesByThread[sample.thread_id].push(sample);
+  }
+
+  for (const key in samplesByThread) {
+    samplesByThread[key].sort((a, b) => a.timestamp - b.timestamp);
+  }
+
+  const profiles: Profile[] = [];
+
+  for (const key in samplesByThread) {
+    const profile: Profiling.ContinuousProfile = {
+      ...input,
+      ...input.profile,
+      samples: samplesByThread[key],
+    };
+
+    profiles.push(
+      wrapWithSpan(
+        options.span,
+        () => ContinuousProfile.FromProfile(profile, frameIndex),
+        {
+          op: 'profile.import',
+          description: 'sampled',
```

Comment:
Can we give this a different description, perhaps `continuous`?

Reply:
I thought that since it's sampled we could just leave it, but better to distinguish this since these might be a lot longer than the old sampled profiles